src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -1,18 +1,32 @@
 import * as React from "react"
 import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
 
+import { RecordSourceSelectorProxy, SelectorData } from "relay-runtime";
 import { ContextConsumer, ContextProps } from "../../../Artsy"
+import { ItemLinkUpdater } from "./ItemLink";
 import SelectableItemContainer from "./SelectableItemContainer"
 
 export interface RelayProps {
+  term: string
   viewer: {
     match_artist: any[]
   }
 }
 
-class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
+class ArtistSearchResultsContent extends React.Component<RelayProps, null> implements ItemLinkUpdater {
```

Comment:
Oooh, very interesting use of `implements`!

However, it’s more idiomatic in JS (and our codebases) to pass just a function rather than an object that has a specific method.

Reply:
One nice thing about `implements Interface` is you can let the editor generate a method stub (at least in IntelliJ) and you don't have to deal with `.bind(this)`.